File: RetaguardaESilva/BackEnd/src/RetaguardaESilva.Domain/Models/Produto.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RetaguardaESilva.Domain.Models
{
    public class Produto
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public int Quantidade { get; set; }
        public bool Ativo { get; set; }        
        public decimal Preco { get; set; }
        public double Codigo { get; set; }
        public DateTime? DataCadastroProduto { get; set; }  
        public int EmpresaId { get; set; }
        public int FornecedorId { get; set; }  
    }
}

File: RetaguardaESilva/FrontEnd/RetagurdaESilva-App/src/app/models/produto.ts
export interface Produto {
    id:number;
    nome:string;
    quantidade:number;
    quantidadeVenda:number;
    ativo:boolean;
    precoCompra:number;
    precoVenda:number;
    precoCompraFormatado:string;
    precoVendaFormatado:string;
    codigo:number;
    dataCadastroProduto:string;
    empresaId:number;
    fornecedorId:number;
}
